source/_integrations/moehlenhoff_alpha2.markdown

```diff
@@ -0,0 +1,54 @@
+---
+title: Möhlenhoff Alpha 2
+description: Instructions on how to integrate a Möhlenhoff Alpha 2 temperature control system into Home Assistant.
+ha_category: Climate
+ha_release: 2022.1
+ha_iot_class: Local Polling
+ha_config_flow: true
+ha_codeowners:
+  - '@j-a-n'
+ha_domain: moehlenhoff_alpha2
+ha_platforms:
+  - climate
+---
+
+The `moehlenhoff_alpha2` platform allows you to control a [Möhlenhoff Alpha 2](https://www.moehlenhoff.de/en/products/room-by-room-control/oem-alpha-2-system) temperature control system.
+
+## Setup
+
+Please make sure the base is turned on and connected to your local network.
```

Comment:
Would `base station` instead of `base` be better?

Reply:
Yes, could be.

Reply:
Changed it.